docs/community/README.md

```diff
@@ -1,40 +1,40 @@
 ---
 title: Community
-description: Become part of the Filecoin community! Learn how here.
+description: Become part of the Filecoin community!
 ---
-#TODO REPLACE OR REMOVE FULL SECTION (only ProtoSchool section on this page has been updated)
+
 # Community
 
-IPFS is home to a vibrant, diverse community of thousands of contributors and participants from all over the globe! Here's how to get help, support, or just hang out with other folks who are helping to build the distributed web.
+Filecoin is home to a vibrant, diverse community of thousands of contributors and miners from all over the globe! Here's how to get help, support, or just hang out with other folks who are helping to build the distributed web.
 
 ## Ways to contribute
 
-No matter your area of specialty or level of expertise, there are many [ways to contribute](/community/contribute/ways-to-contribute/) to IPFS and make a real difference in the IPFS community at large.
+No matter your area of specialty or level of expertise, there are many [ways to contribute](/community/contribute/ways-to-contribute/) to Filecoin and make a real difference in the Filecoin community at large.
 
 ## IPFS forums
 
-The official [IPFS forums](https://discuss.ipfs.io/) are the primary home of IPFS-related discussion. Join the conversation on [https://discuss.ipfs.io/](https://discuss.ipfs.io/) for everything from technical questions, project brainstorming, talk on IPFS use cases and more.
+The official [Filecoin forums](https://discuss.filecoin.io/) are the primary home of Filecoin-related discussion. Join the conversation on [https://discuss.filecoin.io/](https://discuss.filecoin.io/) for everything from technical questions and project brainstorming to discussion of Filecoin use cases and more.
 
 ## IRC
```

Comment:
Matrix/Slack instead of IRC.

Reply:
Isn't Matrix an interface for IRC? Like IRC is to Matrix as email is to Gmail? IPFS has bridges so you access their IRC through whichever IRC client(?) you want - don't know if that's true here or if you can only use Matrix.